Show tasklist sidebar on publication content

Davidslv opened the pull request:
This will show the tasklist sidebar on a specific publication page that
is part of learn to drive.

Trello: https://trello.com/c/WWZn2kNO/

## Before

![before](https://user-images.githubusercontent.com/136777/33025624-65fd8f4a-ce06-11e7-9184-2188f3d5c6f5.png)


## After

![show me tell me questions car driving test gov uk](https://user-images.githubusercontent.com/136777/33025630-69dc8148-ce06-11e7-8183-3c49c381aa2a.png)

andysellick commented on app/views/content_items/publication_with_tasklist_sidebar.html.erb:
```diff
@@ -0,0 +1,42 @@
+    <%= render 'shared/history_notice', content_item: @content_item %>
+    <%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
+
+    <div class="grid-row">
```
This div isn't needed and is pushing its content out of the flow of the page. Grid elements only work if they wrap grid columns, which isn't the case here.

andysellick commented on app/views/content_items/publication_with_tasklist_sidebar.html.erb:
```diff
@@ -0,0 +1,42 @@
+      </div>
+    </div>
+
+    <div class="grid-row">
```
Same here - div not necessary.

andysellick commented on app/views/content_items/publication_with_tasklist_sidebar.html.erb:
```diff
@@ -0,0 +1,42 @@
+
+  <%= render 'shared/sidebar_tasklist', tasklist: tasklist %>
+</div>
+
```
Nitpick - double line space.

andysellick commented on app/views/content_items/publication_with_tasklist_sidebar.html.erb:
```diff
@@ -0,0 +1,42 @@
+</div>
+
+
+<%= render 'shared/footer', @content_item.document_footer %>
```
If the task list is fully expanded it pushes the footer to the bottom of the page, which is a bit odd...

![screen shot 2017-11-21 at 11 19 18](https://user-images.githubusercontent.com/861310/33069880-e03e71b4-cead-11e7-8f0a-523b9f6aaa3b.png)

It could be moved up inside the main grid element and placed at the bottom of the column-two-thirds element to fix this.

Davidslv replied:
@andysellick been addressed

Davidslv commented:
@andysellick I've addressed your comments this is how it looks now:

![after after-sidebar](https://user-images.githubusercontent.com/136777/33075135-6a3e11aa-cec0-11e7-87bb-05e1daf78c8d.png)
